visualizations/frontend/other/TTable.vue

```diff
@@ -1067,11 +1102,34 @@ export default {
     setPagerResetFunction(resetFunction) {
       this.pagerResetFunction = resetFunction;
     },
+    updateFilteredColumns(filterableColumnsToShow) {
+      this.filterableColumnsToShow = filterableColumnsToShow;
+      const thisTable = this.$store.state.layers.components[this.tag_unique];
+      let allValidColumns = [];
+      filterableColumnsToShow.forEach((fcts) => {
+        allValidColumns = allValidColumns.concat(fcts.sqlColumns);
+      });
+      // remove duplicate entries
+      allValidColumns = [...new Set(allValidColumns)];
+      if(allValidColumns.includes(undefined)){
+        console.error('Modify Column Configuration includes "undefined" value, please check that all layer column names are quoted!')
+      }
+      const columnList = JSON.stringify(allValidColumns)
+      thisTable.filters.column_list = columnList;
```

Comment:
I left a comment previously that might have been deleted  by GitHub so I'm posting it back:
This is breaking store immutability. We should create a layer mutation to add column_list to filters or you should set the whole filters object extending it with what you need to add (column_list) using the storeAttribute mutation from the layer store.